### ISSUE TITLE
Add Pie Chart Hover Effect

### ISSUE THREAD

**Issue by GeorgeBerdovskiy** [PureChart/purechart#22]
## Background
PureChart supports very simple pie charts due to #21, but the key word here is _simple_. We would like to make them more interesting by adding elementary interactions. Namely, we would like to highlight a slice when the user hovers over it. Below is a screenshot of what this should look like (taken from [monday.com](https://monday.com/)).

![image](https://github.com/PureChart/purechart/assets/43429820/150748e4-4bdc-4040-8c2b-b9675ecfee92)

After this issue is finished, we'll open another one to add a tooltip displaying the category name and value as well. It's not included here because tooltips are a little more complicated.

## Goal
Add a hover effect to pie chart slices. Basically a slightly lighter / transparent border should appear around the arc of the slice in focus. Again, see the image above for guidance.

Good luck!

**Comment by GeorgeBerdovskiy** [comment]
Note - the data should be formatted like this.
```rb
[
  {
    value: 25,
    color: "#eb4d4b"
  },
  {
    value: 100,
    color: "#be2edd"
  },
  ...
]
```

**Comment by yooian** [comment]
Ill do some more pie

**Comment by GeorgeBerdovskiy** [comment]
Closed by #28
